README.md

```diff
@@ -104,6 +104,39 @@ Terms are compared under the semantics of _toEqualRdfTerm_.
 expect([namedNode('t1'), namedNode('t2')]).toEqualRdfTermArray([namedNode('t2'), namedNode('t3')]);
 ```
 
+#### toBeRdfDatasetContaining
+
+Check if a dataset contains one or more quads.
+
+```js
+const q1 = quad(namedNode('s1'), namedNode('p1'), namedNode('o1'), namedNode('g1'));
+const q2 = quad(namedNode('s2'), namedNode('p2'), namedNode('o2'), namedNode('g2'));
+const d = dataset([q1, q2]);
+expect(d).toBeRdfDatasetContaining(q1, q2);
```

Comment:
Perhaps you can also add `expect(d).toBeRdfDatasetContaining(q1);` here as an example, to show that any number of quads can be entered.